### ISSUE TITLE
version not correct

### ISSUE THREAD

**Issue by JarvanMo** [theyakka/fluro#55]
```
The current Dart SDK version is 2.0.0-dev.58.0.flutter-f981f09760.

Because xx depends on fluro >=1.3.2 which requires SDK version >=2.0.0-dev.61.0 <3.0.0, version solving failed.
pub get failed (1)
```
after I run ```flutter upgrade``` it's still the same.

**Comment by lukef** [comment]
I ran a flutter upgrade now and i get `Dart VM version: 2.0.0-dev.69.4.flutter-08f59e5de3 (Sat Jul 28 00:01:50 2018 +0000) on "linux_x64"`. Is it possible that the upgrade didn't work?

**Comment by lukef** [comment]
I'll drop the version to 58 anyway because it shouldn't matter

**Comment by JarvanMo** [comment]
the upgrade maybe not works.Thanks.

**Comment by lukef** [comment]
I published 1.3.4 to be safe. Give it a little while and then upgrade.

**Comment by JarvanMo** [comment]
nice work.thanks a lot.
